src/Cell.cpp

```diff
@@ -944,6 +1148,15 @@ std::string Cell::toString() {
   else
     string << ", type = UNFILLED";
 
+  if (_rotated) {
```

Comment:
Actually, I'm leaving out the line breaks since we haven't used line breaks in any of our `toString()` methods. I do think it would be nice to clean up the strings output from these methods with line breaks in a consistent manner at some point in the future.